core/psa_crypto.c

```diff
@@ -493,6 +494,19 @@ psa_status_t mbedtls_to_psa_error(int ret)
 #endif
 #endif
 
+#if defined(MBEDTLS_LMS_C)
+        case MBEDTLS_ERR_LMS_BAD_INPUT_DATA:
+            return PSA_ERROR_INVALID_ARGUMENT;
+        case MBEDTLS_ERR_LMS_OUT_OF_PRIVATE_KEYS:
```

Comment:
“Out of private keys” is a rather distinctive error. I'd rather not unify it with a PSA error that only vaguely matches it, in case PSA ends up standardizing LMS signatures but picking a different error code. So it should remain untranslated.

In any case, it won't be raised by PSA functions, since PSA would only provide LMS verification, not LMS signing.